logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/AssetContent.kt

```diff
@@ -9,6 +11,28 @@ data class AssetContent(
     val uploadStatus: Message.UploadStatus = Message.UploadStatus.NOT_UPLOADED,
     val downloadStatus: Message.DownloadStatus
 ) {
+
+    private val isPreviewMessage = sizeInBytes > 0 && !hasValidRemoteData()
+
+    private val isValidAsset =
+        when (metadata) {
+            // in some cases back-end returns no width and height
+            // in that case we do not want to show the asset
+            // and mark it as invalid
+            is AssetMetadata.Image ->
+                metadata.width > 0 && metadata.height > 0
+
+            is AssetMetadata.Audio ->
+                true
```

Comment:
Same here, probably marking this as false will do the job of handling/not handling audio messages. But if it is out of the scope would be better to discard this for the moment